lib/openapi/revisions/est-spec-1.0.0-20231214.json

```diff
@@ -0,0 +1,1536 @@
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Error metrics retrieved successfully",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ErrorMetrics"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid date parameters"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      }
+    },
+    "/api/est/v1/error/monthly": {
+      "get": {
+        "tags": [
+          "Error Metrics"
+        ],
+        "description": "Get monthly error metrics based on start and end date-time parameters",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "X-PhotonIQ-CustomerID",
+            "schema": {
+              "type": "string"
+            },
+            "required": true,
+            "description": "PhotonIQEST CustomerID"
+          },
+          {
+            "in": "query",
+            "name": "from",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "required": true,
+            "description": "The UTC start date and time when error information is retrieved from (e.g. 2022-01-01T00:00:00Z)"
+          },
+          {
+            "in": "query",
+            "name": "to",
+            "schema": {
+              "type": "string",
+              "format": "date-time"
+            },
+            "description": "The UTC end date and time when error information is retrieved from (e.g. 2022-01-01T00:00:00Z)",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Error metrics retrieved successfully",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/ErrorMetrics"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid date parameters"
+          },
+          "500": {
+            "description": "Internal server error"
+          }
+        }
+      }
+    },
+    "/api/est/v1/settings": {
+      "post": {
+        "tags": [
+          "Settings"
+        ],
+        "description": "Create PhotoniqEST service settings",
+        "responses": {
+          "200": {
+            "description": "Returns created PhotoniqEST service settings"
+          },
+          "500": {
+            "description": "Error creating PhotoniqEST service settings"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Settings"
+        ],
+        "description": "Get PhotoniqEST service settings",
```

Comment:
```suggestion
        "description": "Get PhotonIQ EST settings.",
```

Reply:
This whole section needs more information. What kind of settings are we talking about here? How do I post them? What are they even named?

Reply:
Remove "service" after EST in this section and throughout.